src/System.Management.Automation/engine/hostifaces/HostUtilities.cs

```diff
@@ -772,10 +772,16 @@ internal static string GetRemotePrompt(RemoteRunspace runspace, string basePromp
             {
                 return basePrompt;
             }
-            else
+
+            SSHConnectionInfo sshConnectionInfo = runspace.ConnectionInfo as SSHConnectionInfo;
+            if (sshConnectionInfo != null &&
+                !string.IsNullOrEmpty(sshConnectionInfo.UserName) &&
+                !System.Environment.UserName.Equals(sshConnectionInfo.UserName, StringComparison.OrdinalIgnoreCase))
```

Comment:
Aren't unix usernames case sensitive?

Reply:
You're right, Unix usernames are case-sensitive.  I'll change this to a case sensitive comparison.